src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -262,7 +262,7 @@ public IExternalControl Control
             bool updateDisplayNames = false,
             bool forceUpdateDisplayNames = false,
             IExternalRule rule = null,
-            Features features = Features.None)
+            Features features = Features.DefaultFeatures)
```

Comment:
>DefaultFeatures [](http://example.com/codeflow?start=41&length=15)

Why?
This could break things

Reply:
I know it could break but keeping "None" makes no sense.
The default value should be "DefaultFeatures" which is defined as the PFX10 set (and will evolve over time)